requirements-dev.txt

```diff
@@ -14,6 +14,8 @@ aniso8601==9.0.1
     #   flask-restx
 asserts==0.11.1
     # via -r requirements-dev.in
+atomicwrites==1.4.1
```

Comment:
Can we revert all changes to requirements files? I don't think this PR requires any?

Reply:
hopefully that should all be reverted now?? 🤞🏾